app/services/account_membership_cloner.rb

```diff
@@ -0,0 +1,43 @@
+class AccountMembershipCloner
+
+  PROTECTED_USER_ROLE = "Owner".freeze
```

Comment:
We may want to have this reference other constants, i.e. `PROTECTED_USER_ROLE = AccountUser::ACCOUNT_OWNER`. If that causes circular loading or something, we could use the `AccountUser` constants and not worry about having them here. I don't feel super strongly about it, though.